README.md

```diff
@@ -36,16 +36,18 @@
 
 ## Usage
 
-1. Clone this repository locally
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Download the flavor of your choice from [`themes/`](./themes/).
```

Comment:
Don't think we should reference a non-existent directory

Reply:
Yeah it was an optimistic followup of my Whiskers one. What should I use instead? Should we create a `themes/.gitkeep`?

Reply:
Yeah to be honest, my ideal diff is removing the `Clone this repository locally` step since most of our ports don't require the entire repository to be cloned.

 Since we expect them to delete it anyways, what about quite simple and vague like:

```
1. Download theme
2. Apply theme
3. Enjoy!
```

Reply:
How does it look now?